vivisect/symboliks/emulator.py

```diff
@@ -88,7 +88,7 @@ def addRandomSeed(self):
         by solving for deltas between two symbols in different seed
         inputs to see if they are likely arithmetically related.
         '''
-        self._sym_rseed = binascii.hexlify(os.urandom(10))
+        self._sym_rseed = e_common.hexify(os.urandom(10))
```

Comment:
do we really want this as a string?  or is bytes preferred?

Reply:
I kept it as a string because the declaration in the constructor is a string.

That being said, I can't find any uses of this anywhere, so there's another discussion of what is this even meant for, but that's for another day. This PR is already large enough as it is.

Reply:
agreed.
in the past, this has been useful for analysis modules not included in the codebase.  
since one of the easy things for comparing AST's is comparing their hashes... to doublecheck that it's not a fluke (like you had numerical values that happened to line up), addRandomSeed was used to do a second comparison, with some entropy, to ensure they are the same without manually walking through the trees and doing object-level-comparisons.

i've personally used this in ROP-gadget hunting among other things (in my private collection).

it's been a while since i dusted off a couple of those, i should make an effort to write some unittests and make sure everything is still working with each Viv release.

Reply:
is there a unittest for this? :)